### DISCUSSION TITLE
My text is blurry ，but how to  set the proper pixels_per_point in the input to egui.

### DISCUSSION THREAD

**Discussion by moon989** [emilk/egui#560] [Q&A]
egui web wasm text is blurry ,  how to set the proper pixels_per_point , or there is  any other way  ？

**Comment by parasyte** [comment]
`egui_web` uses [`window.devicePixelRatio`](https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio) to select the correct `pixels_per_point` value for you. There shouldn't be anything extra needed.

Are you doing anything particularly different from the demo app?
